src/pages/networks/forms/NetworkDefaultACLReadOnly.tsx

```diff
@@ -0,0 +1,51 @@
+import type { FormikProps } from "formik";
+import type { FC } from "react";
+import type { NetworkFormValues } from "./NetworkForm";
+import { Icon } from "@canonical/react-components";
+
+const NetworkDefaultACLReadOnly: FC<{
```

Comment:
The default case is broken. We should state the correct default if the keys are missing.

<img width="811" height="85" alt="image" src="https://github.com/user-attachments/assets/10bb90d0-85e5-4282-8c37-ca34e6970590" />